tests/apps/sequencing_metrics_parser/test_sequencing_metrics_calculator.py

```diff
@@ -1,32 +1,40 @@
+import pytest
+
 from cg.apps.sequencing_metrics_parser.sequencing_metrics_calculator import (
     calculate_q30_bases_percentage,
     calculate_average_quality_score,
 )
 
 
-def test_calculate_bases_with_q30_percentage():
+@pytest.mark.parametrize(
+    "q30_yield, total_yield, expected_result", [(100, 200, 100 / 200 * 100), (0, 0, 0)]
```

Comment:
Python is smart enough to know that `100/200*100 == (100/200)*100` That was the order in the functions it was testing, so that's why i kept it. Would you prefer parentheses?

Reply:
I always like the added clarity, but feel free to resolve without changing it if it is not worth the time.